src/scancode/cli.py

```diff
@@ -442,7 +443,14 @@ def scan_event(item):
 
     files_scanned_per_second = round(float(files_count) / scanning_time , 2)
     scan_summary['files_scanned_per_second'] = files_scanned_per_second
-
+    
+    if not scanning_errors:
+        #Log file containing timestamp & Project's name.
+        log = open (input_path + "_log.txt", "a+")
```

Comment:
you should use a `with` statement (to close the file). You are leaking an open file